packages/dataviews/src/components/dataviews-filters/search-widget.tsx

```diff
@@ -101,9 +101,10 @@ function ListBox( { view, filter, onChangeView }: SearchWidgetProps ) {
 				__( 'List of: %1$s' ),
 				filter.name
 			) }
-			// @ts-expect-error check if this actually works, since `onFocusVisible`
-			// only works when `focusable` is `true`.
 			onFocusVisible={ () => {
+				// `onFocusVisible` needs the `Composite` component to be focusable,
+				// which is implicitly achieved via the `virtualFocus: true` option
+				// in the `useCompositeStore` hook.
```

Comment:
👍

Reply:
@diegohaz can you confirm that my conclusions are correct? Is this the expected behavior? Should we pass `focusable` to `Composite` to future-proof this behavior?

Reply:
The description makes sense.

> Should we pass `focusable` to `Composite` to future-proof this behavior?

Not all composite containers are focusable, though. For example, a toolbar container is not focusable and therefore doesn't trigger `onFocusVisible`.

Reply:
You can think of `focusable`/`onFocusVisible` as similar to `draggable`/`onDragStart`.

Reply:
> > Should we pass `focusable` to `Composite` to future-proof this behavior?
> 
> Not all composite containers are focusable, though.

That makes sense! Although my question was specific to this instance of `Composite`. My comment is hopefully enough to explain why `onFocusVisible` is being called despite the `focusable` prop is not set on the component.